guides/common/modules/proc_configuring-email-notification-preferences.adoc

```diff
@@ -3,8 +3,10 @@
 [id="Configuring_Email_Notification_Preferences_{context}"]
 = Configuring email notification preferences
 
+[role="_abstract"]
 You can configure {Project} to send email messages to individual users registered to {Project}.
 {Project} sends the email to the email address that has been added to the account, if present.
+
 Users can edit the email address by clicking on their name in the top-right of the {ProjectWebUI} and selecting **My account**.
 
 Configure email notifications for a user from the {ProjectWebUI}.
```

Comment:
```suggestion
```

slightly out of scope but it now feels like an introduction that comes too late. I suggest to drop it because it basically repeats the heading.